### DISCUSSION TITLE
Options status

### DISCUSSION THREAD

**Discussion by themangoest** [eh2k/squares-and-circles#94] [Ideas]
Hello,

At this moment I don't think we can know in what status the Teensy is programmed.
For example:

1.  is it configured for DAC 8564 or 8565
2. screen 1106 or 1306
3. Display SPI enabled or not

Is this possible to have as a feature?

Thanks

**Comment by eh2k** [comment]
@themangoest  

Thanks for the suggestion - I have been preparing the option to read the settings for some time, but never got round to using this function. checkout the webflasher - now you can see the current settings. 

![image](https://github.com/user-attachments/assets/0abcea15-e397-4123-a3c8-7215a7d62e0e)

**Reply by themangoest** [reply]
That is great. Thank you
